### PR TITLE
add new themes

### PR DESCRIPTION
**Pull request by rishav12s**
being playing around the theme presets and i think achieved some good results 

let me know if there are good enough to be merged 

Color Schemes : 

### Everforest
240816_18h04m24s_screenshot

### Monochrome
240816_18h03m03s_screenshot

### One Dark
240816_23h05m49s_screenshot

### Cyberpunk
240816_22h11m49s_screenshot

### CODE REVIEW

**Comment by Jas-SinghFSU** [comment]
Damn, those are some nice themes. I'll test them momentarily.

**Comment by idea90** [comment]
merge please

**Review by idea90 — APPROVED** [review]

**Comment by Jas-SinghFSU** [comment]
@rishav12s  Firstly, I had a chance to go through these... wonderful work.

I add some fixes in my branch `patch/139` which mostly fixes the monochrome theme. It still had colors in the dashboard and a few other spots.

The rest of the themes just required extremely small touchups but they're all good now.

If you can merge that branch into yours then we should be all good to get this PR in the codebase.

**Comment by Jas-SinghFSU** [comment]
Perfect, thank you. I'll repull, retest and merge.

**Comment by rishav12s** [comment]
@Jas-SinghFSU merged it the the monochrome one looks even better now thanks

**Comment by Jas-SinghFSU** [comment]
Was the .gitignore update accidental? I can remove it if so.

**Comment by rishav12s** [comment]
yea the link file was getting pushed you can remove it if you want

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .weather.json
+HyprPanel
```
Comment (Jas-SinghFSU) [inline .gitignore:2]:
```suggestion
```

**Comment by Jas-SinghFSU** [comment]
Thank you for the contribution :heart:

**Pull request merged**